tests/tests/classes/Service.js

```diff
@@ -0,0 +1,384 @@
+'use strict';
+
+/**
+ * Test: Service Class
+ */
+
+const path = require('path');
+const os = require('os');
+const YAML = require('js-yaml');
+const expect = require('chai').expect;
+const Service = require('../../../lib/classes/Service');
+const SError = require('../../../lib/classes/Error');
+const Utils = require('../../../lib/classes/Utils');
+const Serverless = require('../../../lib/Serverless');
+
+describe('Service', () => {
+
+  after((done) => {
+    done();
+  });
+
+  describe('#constructor()', () => {
+    const S = new Serverless();
+
+    it('should attach serverless instance', () => {
+      const serviceInstance = new Service(S);
+      expect(typeof serviceInstance.S._version).to.be.equal('string');
+    });
+
+    it('should construct with defaults', () => {
```

Comment:
So generally I agree with you Philipp, but in this special case I would vote for keeping it in one test. Basically the whole test is "make sure the defaults are set" so I think here it makes sense to keep the expects together as they are all going towards a very specific case and splitting them up creates a lot of boilerplate.